### ISSUE TITLE
Sensors sv_* not available after updating Home Assistant Core

### ISSUE THREAD

**Issue by LinusHoppe** [SukramJ/hahomematic#438]
**custom_component/hahomematic version (if applicable):**
1.3.1 via HACS

**Home Assistant version (if applicable):**
Home Assistant Core 2022.5.4
Home Assistant Supervisor 2022.05.1
Home Assistant OS 7.6

**CCU version:**
RaspberryMatic 3.63.9.20220430 as HomeAssitant-Addon

**Problem-relevant configuration:**

Do you use tls? No
Do you use callback? No
Do you use username and password? Yes
Which interfaces do you use? HM-MOD-RPI-PCB/GPIO

**Describe the bug**
I use `sensor.homematic_sv_alarmmeldungen`, `sensor.homematic_sv_dutycycle` and `sensor.homematic_sv_servicemeldungen` to monitor the status of my Homematic environment.
After the last recent Homeassistant Core Update, all 3 sensors are not available.

**To Reproduce**
1. perform Homeassistant Core Update, started on 2022-05-13T04:16:37.542Z
2. After Homeassistant starts, the three sensors are "unavailable". Last sensor change on 2022-05-13T04:20:00.633Z

**Screenshots (if applicable)**
![sensor status](https://user-images.githubusercontent.com/32911160/168236451-39264dfa-b616-49e0-ab5e-1db08a5103ba.png)
![raspberrymatic](https://user-images.githubusercontent.com/32911160/168236461-4c7d2215-fbe3-4122-bdec-2ce37235be4c.png)

**Relevant HA log entries (full log)**
```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
2022-05-13 07:16:19 WARNING (SyncWorker_5) [homeassistant.loader] We found a custom integration homematicip_local which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-05-13 07:17:29 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: homematicip_local
2022-05-13 07:17:30 WARNING (MainThread) [hahomematic.client] set_value failed with ProxyException [(<Fault -1: 'Generic error (TIMEOUT)'>,)]: 000C9D89B1E660:1, SET_POINT_TEMPERATURE, 18.0
2022-05-13 07:17:30 WARNING (MainThread) [hahomematic.client] set_value failed with ProxyException [(<Fault -1: 'Generic error (TIMEOUT)'>,)]: 000C9D89B1F4E4:1, SET_POINT_TEMPERATURE, 18.0
2022-05-13 07:17:31 WARNING (MainThread) [hahomematic.client] set_value failed with ProxyException [(<Fault -1: 'Generic error (TIMEOUT)'>,)]: 000C9D89B1E622:1, SET_POINT_TEMPERATURE, 18.0
2022-05-13 07:17:32 WARNING (MainThread) [hahomematic.client] set_value failed with ProxyException [(<Fault -1: 'Generic error (TIMEOUT)'>,)]: 000C9D89B1F4AB:1, SET_POINT_TEMPERATURE, 18.0
2022-05-13 07:17:35 WARNING (MainThread) [hahomematic.client] set_value failed with ProxyException [(<Fault -1: 'Generic error (TIMEOUT)'>,)]: 000C9D89B1E660:1, SET_POINT_TEMPERATURE, 18.0
2022-05-13 07:17:36 WARNING (MainThread) [hahomematic.client] set_value failed with ProxyException [(<Fault -1: 'Generic error (TIMEOUT)'>,)]: 000C9D89B1F4DF:1, SET_POINT_TEMPERATURE, 18.0
2022-05-13 07:17:36 WARNING (MainThread) [hahomematic.client] set_value failed with ProxyException [(<Fault -1: 'Generic error (TIMEOUT)'>,)]: 000C9D89B1E622:1, SET_POINT_TEMPERATURE, 18.0
2022-05-13 07:17:37 WARNING (MainThread) [hahomematic.client] set_value failed with ProxyException [(<Fault -1: 'Generic error (TIMEOUT)'>,)]: 000C9D89B1F4A4:1, SET_POINT_TEMPERATURE, 18.0
```

**RaspberryMatic logs**
```
Mounting /data as /usr/local (Home Assistant Add-On): OK
Starting watchdog...
Identifying host system: oci, OK
Initializing RTC Clock: no hardware found
Running sysctl: OK
Checking for Factory Reset: not required
Checking for Backup Restore: not required
Initializing System: OK
Starting logging: OK
Init onboard LEDs: init, OK
Starting irqbalance: OK
Starting iptables: OK
Starting network: eth0: link up, fixed, firewall, inet up, 172.30.33.0, OK
Identifying Homematic RF-Hardware: ....HmRF: HM-MOD-RPI-PCB/GPIO, HmIP: HM-MOD-RPI-PCB/GPIO, OK
Updating Homematic RF-Hardware: HM-MOD-RPI-PCB: 2.8.6, not necessary, OK
Starting hs485dLoader: disabled
Starting xinetd: OK
Starting eq3configd: OK
Starting lighttpd: OK
Starting ser2net: disabled
Starting ssdpd: OK
Starting ha-proxy: OK
Starting NUT services: disabled
Initializing Third-Party Addons: OK
Starting LGWFirmwareUpdate: ...OK
Setting LAN Gateway keys: OK
Starting hs485d: disabled
Starting multimacd: .OK
Starting rfd: .OK
Starting HMIPServer: ............OK
Starting ReGaHss: .OK
Starting CloudMatic: OK
Starting NeoServer: OK
Starting Third-Party Addons: OK
Starting crond: OK
Setup onboard LEDs: booted, OK
Finished Boot: 3.63.9.20220430 (raspmatic_oci_arm64)
```

**Diagnostic Information (available on every device)**
```
{
  "home_assistant": {
    "installation_type": "Home Assistant OS",
    "version": "2022.5.4",
    "dev": false,
    "hassio": true,
    "virtualenv": false,
    "python_version": "3.9.9",
    "docker": true,
    "arch": "aarch64",
    "timezone": "Europe/Berlin",
    "os_name": "Linux",
    "os_version": "5.10.103-v8",
    "supervisor": "2022.05.1",
    "host_os": "Home Assistant OS 7.6",
    "docker_version": "20.10.9",
    "chassis": "embedded",
    "run_as_root": true
  },
  "custom_components": {
    "hacs": {
      "version": "1.24.5",
      "requirements": [
        "aiogithubapi>=21.11.0"
      ]
    },
    "homematicip_local": {
      "version": "1.3.1",
      "requirements": [
        "hahomematic==1.3.0"
      ]
    }
  },
  "integration_manifest": {
    "domain": "homematicip_local",
    "name": "Homematic(IP) Local",
    "config_flow": true,
    "documentation": "https://github.com/danielperna84/custom_homematic",
    "issue_tracker": "https://github.com/danielperna84/hahomematic/issues",
    "requirements": [
      "hahomematic==1.3.0"
    ],
    "ssdp": [
      {
        "manufacturer": "EQ3",
        "manufacturerURL": "http://www.homematic.com"
      }
    ],
    "zeroconf": [],
    "homekit": {},
    "dependencies": [],
    "codeowners": [
      "@danielperna84",
      "@SukramJ"
    ],
    "iot_class": "local_push",
    "loggers": [
      "hahomematic"
    ],
    "version": "1.3.1",
    "is_built_in": false
  },
  "data": {
    "config": {
      "entry_id": "81acd35b5879e99e0d1392cdc49bf213",
      "version": 1,
      "domain": "homematicip_local",
      "title": "Homematic",
      "data": {
        "instance_name": "Homematic",
        "host": "de838cd8-raspberrymatic",
        "username": "**REDACTED**",
        "password": "**REDACTED**",
        "callback_host": "0.0.0.0",
        "callback_port": 0,
        "json_port": null,
        "tls": false,
        "verify_tls": false,
        "interface": {
          "HmIP-RF": {
            "port": 2010
          }
        }
      },
      "options": {},
      "pref_disable_new_entities": false,
      "pref_disable_polling": false,
      "source": "user",
      "unique_id": "172.30.33.0",
      "disabled_by": null
    },
    "platform_stats": {
      "binary_sensor": 13,
      "climate": 14,
      "cover": 12,
      "sensor": 40
    },
    "devices": [
      "HmIP-BROLL",
      "HmIP-BWTH",
      "HmIP-SWSD"
    ]
  }
}
```

**Expected behavior**
The sensor data should be available after upgrading or restarting Homeassistant.

**Comment by LinusHoppe** [comment]
I restarted Home Assistant after adding additional logging information to my config, not it works, sensors are back with data again just after having Home assistant started again.

**Comment by SukramJ** [comment]
Hi,
are you running HA and HM-Addon on a Raspberry PI? 
Version 3 or 4 (which model)?

**Comment by LinusHoppe** [comment]
> Hi, are you running HA and HM-Addon on a Raspberry PI? Version 3 or 4 (which model)?

Yes, Raspberry Pi 4

**Comment by SukramJ** [comment]
I think the issue is related to the speed of your HW.
This problem as been reported some times, but as far as i know related to slower HW and an update of HA.

Next release will raise some timeouts.

For future investigations an updated logging would be helpful.
Please add this to your `configuration.yaml`:

```
logger:
  logs:
    hahomematic: info
    custom_components.homematicip_local: info
```

`info` is less noisy then `debug` and should give some more hints, if the problem reoccurs in future.

Please close this issue for now, and reopen it, if this issue reoccurs again.

**Comment by SukramJ** [comment]
I released a new version (1.3.2) with increased timeouts.

**Comment by LinusHoppe** [comment]
Logging set to info already. I've updated to 1.3.2 and did a complete host reboot (which might be the same scenario as HA Core Update). Started properly with all sensor data available.
I'll reopen if this issue reoccurs in the future.
